src/OneExplorer/OneExplorer.ts

```diff
@@ -465,38 +463,53 @@ input_path=${modelName}.${extName}
     };
 
     // Get the list of all the cfg files inside workspace root
-    const confs =
-        readdirSyncRecursive(this.workspaceRoot!.fsPath).filter(val => val.endsWith('.cfg'));
+    const cfgList = readdirSyncRecursive(root).filter(val => val.endsWith('.cfg'));
 
-    for (const conf of confs) {
-      const cfgObj = ConfigObj.createConfigObj(vscode.Uri.file(conf));
+    return cfgList;
+  }
+
+  /**
+   * Search corresponding .cfg files inside the workspace
+   * for the given baseModelNode
+   *
+   * @param baseModelNode a Node of the base model
+   */
+  private searchConfig(baseModelNode: Node) {
+    console.assert(baseModelNode.type === NodeType.baseModel);
+
+    const cfgList = this.getCfgList();
```

Comment:
(1) Introduce getCfgList() helper function

Let's extract out the logic as a function to make it easy to read